DOCS-1981: Adjustment of the chargeback handling fee information (#1061)
MultiSafepay/docs: content/faq/chargebacks/what-is-a-chargeback.md

### Why do you charge a chargeback handling fee?

> If a customer has initiated a chargeback, MultiSafepay will charge a fee to handle the dispute between you and the customer. Visa and Mastercard, as well as the processing bank can also charge fees if the dispute reaches the final stages of where an agreement cannot be reached between you and the customer.<br> 

Reviewer: ```suggestion
If a customer has initiated a chargeback, MultiSafepay will charge a fee to handle the dispute between you and the customer. Visa and Mastercard, as well as the processing bank, may also charge fees if the dispute cannot be settled between you and the customer.<br> 
```
